packages/react-native-platform-override/src/Manifest.ts

```diff
@@ -5,183 +5,91 @@
  * @format
  */
 
-import * as ManifestData from './ManifestData';
+import * as Serialized from './Serialized';
 
 import * as _ from 'lodash';
-import * as crypto from 'crypto';
-import * as path from 'path';
 
+import Override, {deserializeOverride} from './Override';
```

Comment:
curious why not import the interface named like the function. Is that not allowed when it is the default export?

Reply:
Yep, that's exactly it. Default imports go outside of braces and the name you give it doesn't actually matter.